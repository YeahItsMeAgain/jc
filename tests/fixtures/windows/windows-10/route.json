[{"interfaces":[{"id":"8","mac":"00:ff:58:60:5f:61","name":"TAP-Windows Adapter V9"},{"id":"52","mac":"00:15:5d:fd:0d:45","name":"Hyper-V Virtual Ethernet Adapter"},{"id":"64","mac":"00:15:5d:14:4d:b0","name":"Hyper-V Virtual Ethernet Adapter #3"},{"id":"22","mac":"00:50:56:c0:00:01","name":"VMware Virtual Ethernet Adapter for VMnet1"},{"id":"11","mac":"00:50:56:c0:00:08","name":"VMware Virtual Ethernet Adapter for VMnet8"},{"id":"14","mac":"00:50:56:c0:00:02","name":"VMware Virtual Ethernet Adapter for VMnet2"},{"id":"21","mac":"00:50:56:c0:00:0b","name":"VMware Virtual Ethernet Adapter for VMnet11"},{"id":"1","mac":"","name":"Software Loopback Interface 1"},{"id":"57","mac":"00:15:5d:0e:f1:0f","name":"Hyper-V Virtual Ethernet Adapter #2"}]},{"destination":"0.0.0.0","genmask":"0.0.0.0","gateway":"192.168.31.1","iface":"192.168.31.221","metric":30},{"destination":"127.0.0.0","genmask":"255.0.0.0","gateway":"On-link","iface":"127.0.0.1","metric":331},{"destination":"127.0.0.1","genmask":"255.255.255.255","gateway":"On-link","iface":"127.0.0.1","metric":331},{"destination":"127.255.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"127.0.0.1","metric":331},{"destination":"157.0.0.0","genmask":"255.0.0.0","gateway":"157.55.80.1","iface":"127.0.0.1","metric":78},{"destination":"172.17.96.0","genmask":"255.255.240.0","gateway":"On-link","iface":"172.17.96.1","metric":5256},{"destination":"172.17.96.1","genmask":"255.255.255.255","gateway":"On-link","iface":"172.17.96.1","metric":5256},{"destination":"172.17.111.255","genmask":"255.255.255.255","gateway":"On-link","iface":"172.17.96.1","metric":5256},{"destination":"172.26.240.0","genmask":"255.255.240.0","gateway":"On-link","iface":"172.26.240.1","metric":271},{"destination":"172.26.240.1","genmask":"255.255.255.255","gateway":"On-link","iface":"172.26.240.1","metric":271},{"destination":"172.26.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"172.26.240.1","metric":271},{"destination":"172.27.128.0","genmask":"255.255.240.0","gateway":"On-link","iface":"172.27.128.1","metric":271},{"destination":"172.27.128.1","genmask":"255.255.255.255","gateway":"On-link","iface":"172.27.128.1","metric":271},{"destination":"172.27.143.255","genmask":"255.255.255.255","gateway":"On-link","iface":"172.27.128.1","metric":271},{"destination":"192.168.24.0","genmask":"255.255.255.0","gateway":"On-link","iface":"192.168.24.1","metric":291},{"destination":"192.168.24.1","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.24.1","metric":291},{"destination":"192.168.24.255","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.24.1","metric":291},{"destination":"192.168.31.0","genmask":"255.255.255.0","gateway":"On-link","iface":"192.168.31.221","metric":286},{"destination":"192.168.31.221","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.31.221","metric":286},{"destination":"192.168.31.255","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.31.221","metric":286},{"destination":"192.168.216.0","genmask":"255.255.255.0","gateway":"On-link","iface":"192.168.216.1","metric":291},{"destination":"192.168.216.1","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.216.1","metric":291},{"destination":"192.168.216.255","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.216.1","metric":291},{"destination":"192.168.234.0","genmask":"255.255.255.0","gateway":"On-link","iface":"192.168.234.1","metric":291},{"destination":"192.168.234.1","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.234.1","metric":291},{"destination":"192.168.234.255","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.234.1","metric":291},{"destination":"192.168.255.0","genmask":"255.255.255.0","gateway":"On-link","iface":"192.168.255.1","metric":291},{"destination":"192.168.255.1","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.255.1","metric":291},{"destination":"192.168.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.255.1","metric":291},{"destination":"224.0.0.0","genmask":"240.0.0.0","gateway":"On-link","iface":"127.0.0.1","metric":331},{"destination":"224.0.0.0","genmask":"240.0.0.0","gateway":"On-link","iface":"172.17.96.1","metric":5256},{"destination":"224.0.0.0","genmask":"240.0.0.0","gateway":"On-link","iface":"192.168.234.1","metric":291},{"destination":"224.0.0.0","genmask":"240.0.0.0","gateway":"On-link","iface":"192.168.255.1","metric":291},{"destination":"224.0.0.0","genmask":"240.0.0.0","gateway":"On-link","iface":"192.168.24.1","metric":291},{"destination":"224.0.0.0","genmask":"240.0.0.0","gateway":"On-link","iface":"192.168.216.1","metric":291},{"destination":"224.0.0.0","genmask":"240.0.0.0","gateway":"On-link","iface":"192.168.31.221","metric":286},{"destination":"224.0.0.0","genmask":"240.0.0.0","gateway":"On-link","iface":"172.26.240.1","metric":271},{"destination":"224.0.0.0","genmask":"240.0.0.0","gateway":"On-link","iface":"172.27.128.1","metric":271},{"destination":"255.255.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"127.0.0.1","metric":331},{"destination":"255.255.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"172.17.96.1","metric":5256},{"destination":"255.255.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.234.1","metric":291},{"destination":"255.255.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.255.1","metric":291},{"destination":"255.255.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.24.1","metric":291},{"destination":"255.255.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.216.1","metric":291},{"destination":"255.255.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"192.168.31.221","metric":286},{"destination":"255.255.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"172.26.240.1","metric":271},{"destination":"255.255.255.255","genmask":"255.255.255.255","gateway":"On-link","iface":"172.27.128.1","metric":271},{"network":"157.0.0.0","address":"255.0.0.0","genmask":"157.55.80.1","gateway":"3"},{"iface":"1","metric":331,"destination":"::1/128","gateway":"On-link"},{"iface":"57","metric":5256,"destination":"fe80::/64","gateway":"On-link"},{"iface":"14","metric":291,"destination":"fe80::/64","gateway":"On-link"},{"iface":"11","metric":291,"destination":"fe80::/64","gateway":"On-link"},{"iface":"21","metric":291,"destination":"fe80::/64","gateway":"On-link"},{"iface":"22","metric":291,"destination":"fe80::/64","gateway":"On-link"},{"iface":"9","metric":286,"destination":"fe80::/64","gateway":"On-link"},{"iface":"52","metric":271,"destination":"fe80::/64","gateway":"On-link"},{"iface":"64","metric":271,"destination":"fe80::/64","gateway":"On-link"},{"iface":"9","metric":286,"destination":"fe80::244e:2f1e:2096:62b2/128","gateway":"On-link"},{"iface":"21","metric":291,"destination":"fe80::36de:3306:84b0:6717/128","gateway":"On-link"},{"iface":"52","metric":271,"destination":"fe80::8bd5:17a8:5e7d:9a50/128","gateway":"On-link"},{"iface":"57","metric":5256,"destination":"fe80::a625:de92:858:a42a/128","gateway":"On-link"},{"iface":"22","metric":291,"destination":"fe80::af17:660e:b87c:eb22/128","gateway":"On-link"},{"iface":"11","metric":291,"destination":"fe80::d586:c72e:e178:1147/128","gateway":"On-link"},{"iface":"14","metric":291,"destination":"fe80::d8df:ceee:28ba:aad/128","gateway":"On-link"},{"iface":"64","metric":271,"destination":"fe80::f5d1:8bf7:c410:e325/128","gateway":"On-link"},{"iface":"1","metric":331,"destination":"ff00::/8","gateway":"On-link"},{"iface":"57","metric":5256,"destination":"ff00::/8","gateway":"On-link"},{"iface":"14","metric":291,"destination":"ff00::/8","gateway":"On-link"},{"iface":"11","metric":291,"destination":"ff00::/8","gateway":"On-link"},{"iface":"21","metric":291,"destination":"ff00::/8","gateway":"On-link"},{"iface":"22","metric":291,"destination":"ff00::/8","gateway":"On-link"},{"iface":"9","metric":286,"destination":"ff00::/8","gateway":"On-link"},{"iface":"52","metric":271,"destination":"ff00::/8","gateway":"On-link"},{"iface":"64","metric":271,"destination":"ff00::/8","gateway":"On-link"}]