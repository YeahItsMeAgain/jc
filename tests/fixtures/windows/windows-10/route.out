===========================================================================
Interface List
  8...00 ff 58 60 5f 61 ......TAP-Windows Adapter V9
 52...00 15 5d fd 0d 45 ......Hyper-V Virtual Ethernet Adapter
 64...00 15 5d 14 4d b0 ......Hyper-V Virtual Ethernet Adapter #3
 22...00 50 56 c0 00 01 ......VMware Virtual Ethernet Adapter for VMnet1
 11...00 50 56 c0 00 08 ......VMware Virtual Ethernet Adapter for VMnet8
 14...00 50 56 c0 00 02 ......VMware Virtual Ethernet Adapter for VMnet2
 21...00 50 56 c0 00 0b ......VMware Virtual Ethernet Adapter for VMnet11
  1...........................Software Loopback Interface 1
 57...00 15 5d 0e f1 0f ......Hyper-V Virtual Ethernet Adapter #2
===========================================================================

IPv4 Route Table
===========================================================================
Active Routes:
Network Destination        Netmask          Gateway       Interface  Metric
          0.0.0.0          0.0.0.0     192.168.31.1   192.168.31.221     30
        127.0.0.0        255.0.0.0         On-link         127.0.0.1    331
        127.0.0.1  255.255.255.255         On-link         127.0.0.1    331
  127.255.255.255  255.255.255.255         On-link         127.0.0.1    331
        157.0.0.0        255.0.0.0      157.55.80.1        127.0.0.1     78
      172.17.96.0    255.255.240.0         On-link       172.17.96.1   5256
      172.17.96.1  255.255.255.255         On-link       172.17.96.1   5256
   172.17.111.255  255.255.255.255         On-link       172.17.96.1   5256
     172.26.240.0    255.255.240.0         On-link      172.26.240.1    271
     172.26.240.1  255.255.255.255         On-link      172.26.240.1    271
   172.26.255.255  255.255.255.255         On-link      172.26.240.1    271
     172.27.128.0    255.255.240.0         On-link      172.27.128.1    271
     172.27.128.1  255.255.255.255         On-link      172.27.128.1    271
   172.27.143.255  255.255.255.255         On-link      172.27.128.1    271
     192.168.24.0    255.255.255.0         On-link      192.168.24.1    291
     192.168.24.1  255.255.255.255         On-link      192.168.24.1    291
   192.168.24.255  255.255.255.255         On-link      192.168.24.1    291
     192.168.31.0    255.255.255.0         On-link    192.168.31.221    286
   192.168.31.221  255.255.255.255         On-link    192.168.31.221    286
   192.168.31.255  255.255.255.255         On-link    192.168.31.221    286
    192.168.216.0    255.255.255.0         On-link     192.168.216.1    291
    192.168.216.1  255.255.255.255         On-link     192.168.216.1    291
  192.168.216.255  255.255.255.255         On-link     192.168.216.1    291
    192.168.234.0    255.255.255.0         On-link     192.168.234.1    291
    192.168.234.1  255.255.255.255         On-link     192.168.234.1    291
  192.168.234.255  255.255.255.255         On-link     192.168.234.1    291
    192.168.255.0    255.255.255.0         On-link     192.168.255.1    291
    192.168.255.1  255.255.255.255         On-link     192.168.255.1    291
  192.168.255.255  255.255.255.255         On-link     192.168.255.1    291
        224.0.0.0        240.0.0.0         On-link         127.0.0.1    331
        224.0.0.0        240.0.0.0         On-link       172.17.96.1   5256
        224.0.0.0        240.0.0.0         On-link     192.168.234.1    291
        224.0.0.0        240.0.0.0         On-link     192.168.255.1    291
        224.0.0.0        240.0.0.0         On-link      192.168.24.1    291
        224.0.0.0        240.0.0.0         On-link     192.168.216.1    291
        224.0.0.0        240.0.0.0         On-link    192.168.31.221    286
        224.0.0.0        240.0.0.0         On-link      172.26.240.1    271
        224.0.0.0        240.0.0.0         On-link      172.27.128.1    271
  255.255.255.255  255.255.255.255         On-link         127.0.0.1    331
  255.255.255.255  255.255.255.255         On-link       172.17.96.1   5256
  255.255.255.255  255.255.255.255         On-link     192.168.234.1    291
  255.255.255.255  255.255.255.255         On-link     192.168.255.1    291
  255.255.255.255  255.255.255.255         On-link      192.168.24.1    291
  255.255.255.255  255.255.255.255         On-link     192.168.216.1    291
  255.255.255.255  255.255.255.255         On-link    192.168.31.221    286
  255.255.255.255  255.255.255.255         On-link      172.26.240.1    271
  255.255.255.255  255.255.255.255         On-link      172.27.128.1    271
===========================================================================
Persistent Routes:
  Network Address          Netmask  Gateway Address  Metric
        157.0.0.0        255.0.0.0      157.55.80.1       3
===========================================================================

IPv6 Route Table
===========================================================================
Active Routes:
 If Metric Network Destination      Gateway
  1    331 ::1/128                  On-link
 57   5256 fe80::/64                On-link
 14    291 fe80::/64                On-link
 11    291 fe80::/64                On-link
 21    291 fe80::/64                On-link
 22    291 fe80::/64                On-link
  9    286 fe80::/64                On-link
 52    271 fe80::/64                On-link
 64    271 fe80::/64                On-link
  9    286 fe80::244e:2f1e:2096:62b2/128
                                    On-link
 21    291 fe80::36de:3306:84b0:6717/128
                                    On-link
 52    271 fe80::8bd5:17a8:5e7d:9a50/128
                                    On-link
 57   5256 fe80::a625:de92:858:a42a/128
                                    On-link
 22    291 fe80::af17:660e:b87c:eb22/128
                                    On-link
 11    291 fe80::d586:c72e:e178:1147/128
                                    On-link
 14    291 fe80::d8df:ceee:28ba:aad/128
                                    On-link
 64    271 fe80::f5d1:8bf7:c410:e325/128
                                    On-link
  1    331 ff00::/8                 On-link
 57   5256 ff00::/8                 On-link
 14    291 ff00::/8                 On-link
 11    291 ff00::/8                 On-link
 21    291 ff00::/8                 On-link
 22    291 ff00::/8                 On-link
  9    286 ff00::/8                 On-link
 52    271 ff00::/8                 On-link
 64    271 ff00::/8                 On-link
===========================================================================
Persistent Routes:
  None
